auth/api/http/keys/responses.go

```diff
@@ -76,20 +75,17 @@ func (res revokeKeyRes) Empty() bool {
 }
 
 type retrieveJWKSRes struct {
-	Keys                      []auth.JWK `json:"-"`
-	CacheMaxAge               int        `json:"-"`
-	CacheStaleWhileRevalidate int        `json:"-"`
+	Keys                      []auth.PublicKeyInfo `json:"-"`
+	CacheMaxAge               int                  `json:"-"`
+	CacheStaleWhileRevalidate int                  `json:"-"`
```

Comment:
Is it possible to make it config via ?

Reply:
These are configured using env vars, and returned as headers, so we ignore them in JSON body of response.